### DISCUSSION TITLE
Why category.products is not same as collection.products?

### DISCUSSION THREAD

**Discussion by tiendq** [saleor/saleor#10902] [Q&A]
You could sort and filter with `collection.products` [here](https://docs.saleor.io/docs/3.x/developer/api-reference/objects/collection), but you cannot do the same thing with `category.products` [here](https://docs.saleor.io/docs/3.x/developer/api-reference/objects/category).

Generally a category and a collection are same and so why is their `products` query not same? I think sort and filter are basic and should be supported in all listing queries. It result in I needs two queries to retrieve a category and its products instead of just one query with a collection.

**Comment by peelar** [comment]
Hello @tiendq 👋 

It would indeed make sense to unify it. I will push it to the team and let you know about the result. Thank you 🙌.
